### ISSUE TITLE
⌨ Insufficient contrast - visited links in hero

### ISSUE THREAD

**Issue by petertgiles** [GCTC-NTGC/gc-digital-talent#6899]
## ⌨ Accessibility Issue

This visited links in a hero a quite hard to read.  It's an image so Axe is unable to flag it but I'm guessing it misses the minimum expected contrast ratio.

## 📋 Steps to Reproduce

1. Navigate to /en/applicant/dashboard
2. Click one of the hero links so that it is visited

## 📸 Screenshot

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/6ad48cc6-c76f-41bc-9f99-60dae26769cd)

## ✅ Acceptance Criteria

- [ ] Visited hero link are easily legible
